dev/manual_tests/lib/density.dart

```diff
@@ -376,7 +378,9 @@ class _ControlTile extends StatelessWidget {
         padding: const EdgeInsets.all(8.0),
         child: Column(
           children: <Widget>[
-            Align(alignment: AlignmentDirectional.topStart, child: Text(label, textAlign: TextAlign.start)),
+            Align(
+                alignment: AlignmentDirectional.topStart,
+                child: Text(label, textAlign: TextAlign.start)),
```

Comment:
nit: format (add a trailing comma, un-indent this and previous line by 2, and put the closing ) on the next line?

Reply:
Fixed